src/clj/ataru/virkailija/virkailija_routes.clj

```diff
@@ -744,6 +745,7 @@
           (do (information-request/mass-store
                 (-> (:message-and-subject body)
                     (assoc :recipient-target (:recipient-target body))
+                    (assoc :add-update-link (:add-update-link body))
```

Comment:
Ilmeisesti tälle siis oli toteutus jo, ei vaan APIssa keinoa sanoa?

Reply:
Juuri näin, oli toteutettu jo aiemmin yksittäisen viestin lähetykselle mutta jäi puuttumaan massaviesti-apista